app/src/main/java/com/vanniktech/emoji/sample/MainActivity.java

```diff
@@ -114,11 +114,7 @@
   }
 
   @Override public void onBackPressed() {
-    if (emojiPopup != null && emojiPopup.isShowing()) {
-      emojiPopup.dismiss();
-    } else {
-      super.onBackPressed();
-    }
+    super.onBackPressed();
```

Comment:
If we don't do anything here any longer then we can remove the method completely.